alvr/dashboard/src/steamvr_launcher/linux_steamvr.rs

```diff
@@ -162,8 +172,13 @@ fn linux_gpu_checks(device_infos: &[(&wgpu::Adapter, DeviceInfo)]) {
     let mut vrmonitor_path_written = false;
     if have_igpu {
         if have_nvidia_dgpu {
-            let nv_options = "__GLX_VENDOR_LIBRARY_NAME=nvidia __NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only \
-            VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/nvidia_icd.json";
+            let base_path = "/usr/share/vulkan/icd.d/nvidia_icd";
+            let nvidia_icd_path = if Path::new(&format!("{base_path}.json")).exists() {
+                format!("{base_path}.json")
+            } else {
+                format!("{base_path}.x86_64.json")
+            };
```

Comment:
this should go `if ... .json VK_DRIVER_FILES... else if .x86_64.json ... else __VK_LAYER...`, think that'd be best